.storybook/test-data.js

```diff
@@ -499,6 +499,13 @@ const state = {
         decimals: 18,
       },
     ],
+    tokenBalances: {
+      '0x1': {
+        '0x64a845a5b02460acf8a3d84503b0d68d028b4bb4': {
+          balance: '1000000000000000000',
+        },
+      },
+    },
```

Comment:
This doesn't look like the right shape for token balances state.  Token balances is a map from account address -> chain id -> token address -> hexadecimal balance.

Reply:
An example would be something like:

```
    "tokenBalances": {
      "0x11815c8f0a24245d898211f82e15482459400b1b": {
        "0x1": {
          "0x44Acd3081eaa143812423925C7Bde7c83541F8fb": "0x4487ad",
        },
      }
    }
```

Reply:
done [c0c1117](https://github.com/MetaMask/metamask-extension/pull/28553/commits/c0c1117abaf5fb75930ee4e4bd6a806f2ba3578c)